### PR TITLE
Properly set pipeline version on new pipelines

### PR DESCRIPTION
**Pull request by lresende**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx [hunk 1]
```diff
@@ -578,58 +578,68 @@ export class PipelineEditor extends React.Component<
...
-              this.canvasController.setPipelineFlow(pipelineJson);
-            } else {
-              this.handleClosePipeline();
-            }
-          });
+      if (pipelineJson == null) {
+        // creating new pipeline
+        pipelineJson = this.canvasController.getPipelineFlow();
+        if (Utils.isNewPipeline(pipelineJson)) {
+          pipelineJson.pipelines[0]['app_data']['version'] = 1;
```
Comment (kevin-bates) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx:585]: I expected this to just be baked into the template but I see that's just for creating single-node pipelines for batch notebook processing.  Seems like we should be consistent and add a `'version': 1` field to the template as well.

Comment (kevin-bates) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx:585]: We should use the constant here...
```suggestion
          pipelineJson.pipelines[0]['app_data']['version'] = PIPELINE_CURRENT_VERSION;
```

Comment (lresende) [reply packages/pipeline-editor/src/PipelineEditorWidget.tsx:585]: What do you mean by template?

Comment (kevin-bates) [reply packages/pipeline-editor/src/PipelineEditorWidget.tsx:585]: https://github.com/elyra-ai/elyra/blob/master/packages/pipeline-editor/src/pipeline-template.json

**Pull request merged**